### ISSUE TITLE
VoiceOver users are not notified if a page is reloaded because of a problem (Safari)

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#57]
**Upstream bug:** Feedback Assistant #FB9005166
**Date:** 2021-02-15
**Reported by:** Oliver Byford
**Related to:** https://github.com/alphagov/govuk-frontend/issues/2135

---

### Description

When a page is reloaded because of a problem (for example if the WebKit process crashes), a VoiceOver user does not get auditory feedback to tell them this has happened. The status message that appears ("This web page was reloaded because a problem occurred") is not announced. Focus moves back to the top of the page. Additionally, there is no way to focus the status message (for example to get VoiceOver to read it) or to dismiss it.

I imagine this could be very disorientating for a user who is not able to perceive the screen, especially if for example they were halfway through filling in a form, or were interacting with a single page application that does not retain state.
